### DISCUSSION TITLE
Drag and drop feature in the New Project boards

### DISCUSSION THREAD

**Discussion by apenab** [community/community#45240] [Projects and Issues] [answered]
### Select Topic Area

Bug

### Body

The drag and drop feature in the new project boards in the board layout doesn't work well.

**Comment by shonfeder** [comment]
We are experiencing drag and drop of cards being totally broken, as of approx last week: cannot move lanes, or adjust order by dragging.

**Comment by ocundale** [comment, marked as answer]
![image](https://user-images.githubusercontent.com/1638138/214609411-9fff1b0a-58c8-4942-88aa-bb9795c9f3f5.png)

Looks like you need to re-enable manual sorting as shown in the image above.

Seems a lot of people confused by this (including myself!)  ...probably should have been a notification to let people know of the change.

**Reply by apenab** [reply]
Thx, works, dope!

**Reply by mdonges** [reply]
Does not work for me at all.
Items cannot be dragged any more at all.

**Comment by Gbemisola23** [comment]
I don't know what is happening but the DND ain't working right. It's bouncing back

**Reply by mdonges** [reply]
With me its not moving any more on DnD....
Sorting by manual priority is currently impossible
even the buttons allow only to the top or to the bottom but you cannot increase or decrease position by one!

currently totally unusable

**Reply by mdonges** [reply]
seems to work again since today

**Reply by mdonges** [reply]
back to broken state today...

annoying...

**Comment by thomthom** [comment]
Drag and Drop in Firefox appear to be broken. It works fine in Edge, for the same project.

This used to work fine. :/

**Reply by yuanchen-zhu** [reply]
I want to report the same issue. This happens even when in privacy mode. Chrome works fine.

**Reply by oxstone** [reply]
Does not work on Edge/Firefox for me, DnD was fine on Friday

**Reply by icorbrey** [reply]
Yep same for me

**Reply by jeznorth** [reply]
Bumping this. Same issue here - does not work in Chrome / FF / Edge

**Reply by lbussell** [reply]
+1

**Reply by jayspadie** [reply]
For anyone else who may post on here, can you please provide more details of what you're seeing?  This discussion was originally because of the board being sorted.  That's not always obvious, but it restricts some amount of drag-n-drop options.

If things are "broken" in a different way, please let us know what you're seeing to help us investigate.  Also, please let us know if you see (or don't see) the "Increased Items preview" label next to your project title (as shown below, for example).  Thanks.
<img width="362" alt="image" src="https://github.com/user-attachments/assets/17ee97a5-3a3f-4982-bc36-c4016619ad92" />

**Reply by icorbrey** [reply]
The cursor becomes a grabby hand when hovering over cards, but trying to drag them just selects text on the screen. E.g. this is the result of trying to drag the top card in the lefthand column into one next to it:

![image](https://github.com/user-attachments/assets/18f68cd6-ccfe-4327-89ca-e1b848f88f8e)

I don't believe I'm on the preview.

**Reply by thomthom** [reply]
> For anyone else who may post on here, can you please provide more details of what you're seeing? This discussion was originally because of the board being sorted. That's not always obvious, but it restricts some amount of drag-n-drop options.

My project's Sort By is set to `manual`. And I now see the date for the original issue was quite a bit older. So we're probably seeing something new.

> If things are "broken" in a different way, please let us know what you're seeing to help us investigate. Also, please let us know if you see (or don't see) the "Increased Items preview" label next to your project title (as shown below, for example). Thanks. 

I'm seeing what is being described by other here. When the mouse is over an item in the board it changes to the hand that indicates you can drag it. But as you initiate the drag it ends up selecting the text instead. No errors in the dev console. From observation only it appear that the drag events aren't working correctly any more in Firefox. But it continues to work in Edge. (I've not tested Chrome or Safari)

I have no `"Increased Items preview"` indicators on my projects.

*Edit:*
Tested on macOS Chrome Version 133.0.6943.142 (Official Build) (arm64) - ✅ D&D works.
Tested on macOS Firefox Version 135.0.1 (64-bit) - ✅ D&D works.
Tested on macOS Safari Version 17.6 (19618.3.11.11.5) - ✅ D&D works.
Tested on Windows Firefox Version 135.0.1 (64-bit) - ❌ D&D does not work. (Three different machines, 2x Win11, 1x Win10)
Tested on Windows Chrome Version 133.0.69452.127 (Official Build) (64-bit) ❌ D&D does not work. (Win 10)
Tested on Windows Edge Version 133.0.3065.92 (Official Build) (64-bit) ❌ D&D does not work. (Win 10)
Tested on Windows Edge Version 133.0.3065.92 (Official build) (64-bit) - ✅ D&D works. (2x Win11)

**Reply by lbussell** [reply]
Same symptoms as @icorbrey, not using increased items preview, happens with both manual sort and sorted columns. Microsoft Edge 133.0.3065.82.

**Reply by jayspadie** [reply]
Thanks (everyone) for the details and reporting the issue.  We're looking into it.

**Reply by jayspadie** [reply]
Thanks for the additional details.  We've disabled a feature flag for recent accessibility improvements that *might* have caused a Windows laptop with touchscreen to disable the drag-n-drop.  Can anyone confirm if they are still seeing the issue on their Windows machine?

**Reply by thomthom** [reply]
> Thanks for the additional details. We've disabled a feature flag for accessibility improvements that _might_ have caused a Windows laptop with touchscreen to disable the drag-n-drop. Can anyone confirm if they are still seeing the issue on their Windows machine?

I just tested two of my Windows machines, both work! 👍

However, while one was a Surface Pro 11 tablet with touch screen, the other was a desktop machine with normal two-monitor setup.

**Reply by thomthom** [reply]
That being said, the desktop machine has a Wacom intuos 3 tablet connected (old one, no monitor). But maybe that could also affect that accessibility clash..?

**Reply by jayspadie** [reply]
Awesome!  So maybe it doesn't necessarily need to be a Windows touchscreen device, but Windows 10 may not have been playing nicely with our recent changes that checks for such things.  Thanks so much for confirming that it's working for you now @thomthom.

It does seem related to the recent accessibility improvements, so we will leave that feature flag disabled for now until we determine the cause and can work around it.  If anyone else is still seeing a problem with drag-n-drop (regardless of device) please let us know!

Thanks again.

**Reply by lbussell** [reply]
Confirming that the drag-and-drop functionality is working again

**Reply by thomthom** [reply]
> Awesome! So maybe it doesn't necessarily need to be a Windows touchscreen device, but Windows 10 may not have been playing nicely with our recent changes that checks for such things. Thanks so much for confirming that it's working for you now @thomthom.
> 
> It does seem related to the recent accessibility improvements, so we will leave that feature flag disabled for now until we determine the cause and can work around it. If anyone else is still seeing a problem with drag-n-drop (regardless of device) please let us know!
> 
> Thanks again.

The two machines I tested was my Win11 machines. I've yet not to test my Win10 machine.

**Comment by cacraigucar** [comment]
I see the `Increased items preview`, at the top of my projects page, and I have the same problem as @icorbrey described.  The hand highlights the box I want to move, but it does not allow me to move the box.  It just highlights the text of the other boxes when I try moving it.

**Reply by jayspadie** [reply]
@cacraigucar I assume this is also Firefox?  We're still trying to repro on our end.  Would you mind sharing which version of Firefox you're using?  Thanks.

**Reply by cacraigucar** [reply]
Sorry, I should have specified.  I am seeing this is broken in both Chrome and Edge on two different Windows laptops.  A coworker said she tired it with Firefox (again on a Windows laptop) and it is broken for her as well.

**Reply by cacraigucar** [reply]
Chrome Version 133.0.6943.127 and Microsoft Edge Version 133.0.3065.92 are both broken

**Reply by cacraigucar** [reply]
My coworkers Firefox version is: 135.0.1

**Reply by jayspadie** [reply]
Thanks for confirming that you're seeing it in all browsers.  We're investigating it now.

**Reply by thomthom** [reply]
> @cacraigucar I assume this is also Firefox? We're still trying to repro on our end. Would you mind sharing which version of Firefox you're using? Thanks.

I tested on all browsers I had on all my machines - added details to this comment: https://github.com/orgs/community/discussions/45240#discussioncomment-12380534

The pattern is strange. I see it failing in all my Firefox installations. But I see some of my machines failing with Edge/Chrome but passing on others.

**Reply by cacraigucar** [reply]
I have confirmed that this is now working with both Windows 10 and Windows 11 in both Chrome and Edge.  My coworker confirmed that it is now working in Firefox for her.  

Thank you for the quick response!!!

**Reply by jayspadie** [reply]
Great!  Thank you for confirming, @cacraigucar.  We really appreciate everyone's help here.
